webui/react/src/routes/index.ts

```diff
@@ -49,10 +52,22 @@ export const appRoutes: RouteConfigItem[] = [
     path: '/det/logout',
     title: 'Logout',
   },
+  {
+    exact: true,
+    id: '',
+    path: '/',
```

Comment:
I'm not sure here are these paths relative to  `/det` or are absolute? if they're absolute this should probably be `/det` and maybe `*` below could be `/det/*` but `*` covers that anyway

Reply:
yeah you're right, this is not needed because the `*` case should cover this from the React side.